### PR TITLE
Fix: reformatted env subs. Resolves #33

### LINKED ISSUE #33: Multiple issues regarding env variables

### PR DESCRIPTION
**Pull request by VictorTennekes**
Fixes issues with multiple env vars chained like `$PWD$HOME$PWD` and exported values.

### CODE REVIEW

File: src/utils/replace_env.c [hunk 1]
```diff
@@ -15,22 +15,28 @@
...
-		i++;
-	}
-	free(tmp);
+	if (!src || !to_find || !replace)
+		return (NULL);
+	to_find_len = ft_strlen(to_find);
+	replace_len = ft_strlen(replace);
+	if (!to_find_len)
+		return (NULL);
+	res = zalloc(sizeof(char) * ft_strlen(src)
```
Comment (abeaugustijn) [inline src/utils/replace_env.c]: Could just be malloc right?

**Review by abeaugustijn — CHANGES_REQUESTED** [review]
Looks good

File: src/utils/replace_env.c (hunk 1 above)
Comment (VictorTennekes) [reply src/utils/replace_env.c]: Yap, has been changed :smile:

File: src/builtin/builtin_export.c [hunk 2]
```diff
@@ -25,6 +25,8 @@
 **			identifier is given.
 */
 
+#include <stdio.h>
```
Comment (abeaugustijn) [inline src/builtin/builtin_export.c]: Waarvoor is deze?

**Review by abeaugustijn — CHANGES_REQUESTED** [review]
ziet dr prima uit

File: src/builtin/builtin_export.c (hunk 2 above)
Comment (VictorTennekes) [reply src/builtin/builtin_export.c]: Helemaal nop, slordigheidsfoutje.

Comment (VictorTennekes) [reply src/builtin/builtin_export.c]: Is daar blijven hangen na tests.

**Review by abeaugustijn — APPROVED** [review]
Nice

**Pull request merged**